client/src/api/survey.js

```diff
@@ -0,0 +1,20 @@
+/* export async function getQuestionById(id) {
+  const response = await fetch(`http://localhost:5000/survey/${id}`);
+  const question = await response.json();
+  return question;
+} */
+
+export async function getQuestionDoc(id) {
+  const response = await fetch(`/api/survey/${id}`);
+  const question = await response.json();
+  console.log(question);
+  return question;
+}
+
+/* export async function getDataByID({ collectionName, id }) {
+  const result = await fetch(`/api/${collectionName}/${id}`);
+  const data = await result.json();
+  return data;
+}
+
+ */
```

Comment:
Delete it if you don't need it right now :-).
It's a good practice to have only used code in your main branch / production.
If you need it for later, consider using `git stash`.

Same with other comments in this file.